app/models/user.rb

```diff
@@ -1,6 +1,64 @@
 class User < ActiveRecord::Base
+  has_many :identities
+  validates :email, presence: true
+  validates :password, presence: true
+  validates :sign_in_count, presence: true
+
+  TEMP_EMAIL_PREFIX = 'change@me'
+  TEMP_EMAIL_REGEX = /\Achange@me/
+
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :lockable, :timeoutable, :confirmable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+    :recoverable, :rememberable, :trackable, :validatable, :omniauthable
+
+  validates_format_of :email, :without => TEMP_EMAIL_REGEX, on: :update
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.